.github/workflows/kind.yml

```diff
@@ -127,12 +161,21 @@ jobs:
   # yet.
   artifact-cleanup:
     name: Delete uploaded images
-    needs: [build-policy-recommendation-image, test-e2e-encap]
-    if: ${{ always() && needs.build-policy-recommendation-image.result == 'success' }}
+    needs:
+    - build-policy-recommendation-image
+    - build-clickhouse-monitor-image
+    - test-e2e-encap
+    - test-upgrade-from-N-1
+    if: ${{ always() && (needs.build-policy-recommendation-image.result == 'success' || needs.build-clickhouse-monitor-image.result == 'success') }}
```

Comment:
why using or between policy-recommendation and clickhouse-monitor build results?

Reply:
Because the cleanup job is expected to run if any of the images is successfully built.